### DISCUSSION TITLE
Sound artifacts.

### DISCUSSION THREAD

**Discussion by anycolorsmith** [probonopd/MiniDexed#248] [Q&A] [answered]
Strange thing. After saving the performance, artifacts appear in the sound - crackling or rustling. I2C audio. Reboot helps clear sound.

**Comment by craigyjp** [comment, marked as answer]
Yes this has been documented already for saving performances, I believe a fix is either in the works or actually completed

**Reply by probonopd** [reply]
Should be fixed in the latest build thanks to https://github.com/probonopd/MiniDexed/pull/238.
